packages/core/src/lib/Select/SelectBeta/SelectBeta.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { ISelectProps, KEY, OptionProps } from '../types';
+import styles from './SelectBeta.styles';
+import SelectMenuList from './components/SelectMenuList';
+import SelectedItemsList from './components/SelectedItemsList';
+
+const {
+    ButtonStyledSelect,
+    DivStyledDesc,
+    DivStyledOverlay,
+    DivStyledSelectWrapper,
+    DivStyledWrapper,
+    LabelStyled,
+    TriangleDownIconStyled,
+    TriangleUpIconStyled,
+} = styles;
+
+const SelectBeta: React.FunctionComponent<ISelectProps> = ({
+    defaultOptionIndex,
+    disabled = false,
+    id,
+    isMulti = false,
+    isSearch = true,
+    label = 'Select',
+    max = Infinity,
+    name = 'options',
+    onChange,
+    options = [],
+    placeholder,
+    style,
+    value = [],
+    width = '100%',
+    ...props
+}) => {
+    const {
+        customNoDataText,
+        description,
+        errorMessage,
+        onBlurTraditional,
+        onChangeTraditional,
+        prefixIcon,
+        prefixText,
+        refTraditional,
+        state,
+        traditionalHTML5,
+        validation,
+        ...rest
+    } = props;
+    const triggerRef = useRef<HTMLButtonElement>(null);
+    const [isOpen, setIsOpen] = useState(false);
+
+    function elId(component: string) {
```

Comment:
i'm not against `function() {}` but the rest or most of the kit uses `const functionName = () => {}`
its nice to try and align don't you think?

Reply:
also `elId` sounds like a defined const to me, not the name of a function that will normalise a component ID, maybe consider making the name more friendly for the next dev 🤷‍♂️

Reply:
agreed 💯. 
my late night naming skills are not good 😂

Reply:
Changed the name to elementId (elId was short for this 😆) ✅